src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -108,14 +116,16 @@ def get(self, request, organization, trace_id):
             else:
                 break
         if extra_roots > 0:
+            sentry_sdk.set_tag("discover.trace-view.warning", "root.extra-found")
             logger.warning(
                 "discover.trace-view.root.extra-found",
                 {"extra_roots": extra_roots, **warning_extra},
             )
 
         parent_map = defaultdict(list)
         for item in result["data"]:
-            parent_map[item["trace.parent_span"]].append(item)
+            if not is_root(item):
```

Comment:
Yeah, but wasn't an issue unless we circled back to root (... which I think is impossible since it'd require a span to have no id? 😨 I hope that's impossible)